clang/include/clang/Driver/Options.td

```diff
@@ -6584,6 +6584,10 @@ def fsycl_use_main_file_name : Flag<["-"], "fsycl-use-main-file-name">,
   HelpText<"Tells compiler that -main-file-name contains an absolute path and "
            "file specified there should be used for checksum calculation.">,
   MarshallingInfoFlag<CodeGenOpts<"SYCLUseMainFileName">>;
+def fsycl_allow_virtual_functions : Flag<["-"], "fsycl-allow-virtual-functions">,
+  HelpText<"Allow virtual functions calls in code for SYCL device">,
+  Flags<[CC1Option, NoDriverOption]>,
```

Comment:
This is redundant - already in a section that sets `CC1Option` and `NoDriverOption`

Reply:
Done 5feaff5